packages/cadl-ranch-specs/http/models/collections-basic/main.cadl

```diff
@@ -52,10 +52,10 @@ op sendCollectionModel(@body input: InputModel): Response<200> | ErrorResponse;
 @scenarioDoc("Generate and receive output model with required collection properties.")
 @route("/models")
 @get
-op getCollectionModel(): OkResponse<OutputModel> | ErrorResponse;
+op getCollectionModel(): (OkResponse & Body<OutputModel>) | ErrorResponse;
```

Comment:
```suggestion
op getCollectionModel(): OutputModel | ErrorResponse;
```

Reply:
These two mean the same things, I can update though to keep it consistent